src/components/ui/drawers/categories/SetupCategoryCreationDrawer.tsx

```diff
@@ -6,23 +6,30 @@ import {
 } from '@/constants/schema';
 import { CategoryCreationDrawerContent } from '@/components/ui/drawers/categories/CategoryCreationDrawerContent';
 import { useState } from 'react';
+import { ICategory } from '@/types/commons';
+import { useToast } from '@/components/ui/use-toast';
+// import { randomUUID } from 'crypto';
```

Comment:
Could you delete this import?

Reply:
I changed to `self. crypto. randomUUID()` to use `randomUUID()`